src/Button/OutlineButton.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Button from './Button'
+
+const OutlineButton = styled(Button)`
+  background-color: ${props => props.theme.colors.white};
+  color: ${props => props.color || props.theme.colors.blue};
+  border-width: 2px;
+  border-color: ${props => props.color || props.theme.colors.blue};
```

Comment:
In CSS, `border-color` is the same as the `color` for the element, so this line isn't needed. Same goes for if you were to refactor this with `boxShadow`